pages/admin/content-manager/[entity_type_slug]/[id].js

```diff
@@ -145,8 +145,8 @@ export default function Type({ cache }) {
 
   const getS3Keys = (files) => {
     const fileKeys = Object.keys(files);
-    const s3Keys = fileKeys.map((file) => state.values[file].key);
-
+    const s3Keys = fileKeys.filter((file) => state.values[file].key);
```

Comment:
You can remove the parenthesis if the parameter is only one. 
```suggestion
    const s3Keys = fileKeys.filter(file => state.values[file].key);
```